pkgs/applications/editors/music/tuxguitar/default.nix

```
@@ -1,17 +1,12 @@
 { lib, stdenv, fetchurl, swt, jre, makeWrapper, alsa-lib, jack2, fluidsynth, libpulseaudio }:
 
-let metadata = assert stdenv.hostPlatform.system == "i686-linux" || stdenv.hostPlatform.system == "x86_64-linux";
-  if stdenv.hostPlatform.system == "i686-linux" then
-    { arch = "x86"; sha256 = "sha256-k4FQrt72VNb5FdYMzxskcVhKlvx8MZelUlLCItxDB7c="; }
-  else
-    { arch = "x86_64"; sha256 = "sha256-mj5wVQlY2xFzdulvMdb5Qb5HGwr7RElzIkpOLjaAfGA="; };
-in stdenv.mkDerivation rec {
-  version = "1.5.5";
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```

I think we could use `stdenvNoCC` here. What do you think?